packages/ogmios/src/CardanoNode/OgmiosObservableCardanoNode/OgmiosObservableCardanoNode.ts

```diff
@@ -149,7 +149,7 @@ export class OgmiosObservableCardanoNode implements ObservableCardanoNode {
                   chainSync$: createObservableChainSyncClient(
                     { intersectionPoint: intersection.point },
                     { interactionContext$: this.#interactionContext$ }
-                  ),
+                  ).pipe(ObservableCardanoNode.bufferChainSyncEvent(10)),
```

Comment:
This should be configurable. Adding it as argument to `findIntersect` doesn't seem right, so maybe it's not the best place to apply this operator. `fromCardanoNode` seems like a better place.
```suggestion
                  ).pipe(ObservableCardanoNode.bufferChainSyncEvent(bufferSize)),
```

Reply:
Sorry @mkazlauskas , I can't understand where...
Could you please tell me, precisely, where in `fromCardanoNode` should it be moved?
Thank you

Reply:
I think we can add buffer size prop [here](https://github.com/input-output-hk/cardano-js-sdk/blob/master/packages/projection/src/Bootstrap/fromCardanoNode.ts#L125-L127) and apply the operator [here before withRolledBackBlock](https://github.com/input-output-hk/cardano-js-sdk/blob/master/packages/projection/src/Bootstrap/fromCardanoNode.ts#L44)

Reply:
0a7e9f0dc and 05e82468f should solve this

Reply:
`tsc` error in `projection-typeorm` tests (it's using `fromCardanoNode`). I suggest setting a default in there as well, or just make it optional and `passthrough()` instead of the buffer if not set

Reply:
Why it is not caught by `yarn test:build:verify`? Checking...

Reply:
The change to `tsconfig.json` in dd11eab11 solves the `yarn test:build:verify` problem as well